rolling/distribution.yaml

```diff
@@ -1844,6 +1844,17 @@ repositories:
       url: https://github.com/ros/resource_retriever.git
       version: ros2
     status: maintained
+  rmf_cmake_uncrustify:
+    release:
```

Comment:
@marcoag no `doc` entry?

Reply:
Added in https://github.com/ros/rosdistro/pull/29499/commits/b047f0512db9ef93a5cf871349513f2dfe53dea7
Thanks for checking.